src/players/Vimeo.js

```diff
@@ -6,7 +6,7 @@ import createSinglePlayer from '../singlePlayer'
 const SDK_URL = 'https://player.vimeo.com/api/player.js'
 const SDK_GLOBAL = 'Vimeo'
 const MATCH_URL = /vimeo\.com\/.+/
-const MATCH_FILE_URL = /vimeo\.com\/external\/.+\.mp4/
+const MATCH_FILE_URL = /vimeo\.com\/external\/.+\.[mp4|m3u]/
```

Comment:
Why don't we just look for any extension?
```suggestion
const MATCH_FILE_URL = /vimeo\.com\/external\/[0-9]+\..+/
```

Reply:
Will try to get access to the vimeo Pro account to find out what these urls look like – but you're probably right, there are more extensions than only `mp4` and `m3u`.